src/azul/terraform.py

```diff
@@ -793,18 +793,50 @@ def tf_config(self, app_name):
         del deployment['lifecycle']
         deployment['triggers'] = {'redeployment': deployment.pop('stage_description')}
 
+        # FIXME: Replace with single_quote() from strings.py
```

Comment:
FIXMEs can't refer to PRs. This should be a drop commit and the PRs should be chained. Or you just wait with the review request until the blocking PR has been merged.